internal/retryhttp/roundtripper.go

```diff
@@ -30,7 +30,7 @@ func (rt *RoundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
 
 	resp, err := rt.client.Do(retryableReq)
 	if err != nil && resp != nil && resp.StatusCode == http.StatusForbidden && strings.EqualFold(resp.Header.Get("server"), "cloudfront") {
-		return nil, platform.NewCountryBlockedError()
+		return nil, api_errors.NewCountryBlockedError()
```

Comment:
Yeah, a while ago I had brought this up in stand, but it probably bears repeating. Our codebase has gotten to such a large size that following idiomatic Go naming conventions gives us too many collisions. At which point we're using aliases where needed, and those aren't usually very consistent.

Given the givens I'd rather have underscores and non-colliding package names than no underscores and collision galore. 

Imo the naming conventions only make sense on projects that are Go libraries, once you start working on a Go application it very quickly stops making sense.

Reply:
I agree that the aliases were getting a bit out of hand but I'm still not seeing how underscores solve this problem. Could `api_errors` not be `apiErrors` or does go complain about that as well?

Either way this should be considered a nit as I don't want to hold the PR up. I just wanted to flag us introducing something that we have called out in the past for both file and package names.

Reply:
Oh sure, yeah it could be. Especially now that we're ignoring idiomatic Go in this context anyway. I just used underscore since that at least partly aligns with Go's standard (eg. `pkg_test.go`), plus it sidesteps the whole case sensitive filesystems issue.